goal_src/jak2/engine/gfx/sprite/particles/sparticle-h.gc

```diff
@@ -15,6 +16,7 @@
   (bit2 2)     ;; cleared after an aux has its func set to add-to-sprite-aux-lst
   (bit3 3)
   (bit4 4)
+  (unk5 5)
```

Comment:
unique name here

Reply:
it is unique in the context of this enum? Do you want something globally unique?

Reply:
globally yeah. It makes it way easier to search for usages later when we want to give them normal names.